_dev/src/views/onboarding-page.vue

```diff
@@ -107,15 +107,15 @@
         || merchantCenterAccountConnectedOnce
         || productFeedIsConfiguredOnce
         || freeListingIsActivatedOnce
-        || googleAdsAccountConnectedOnce
+        || (googleAdsAccountConnectedOnce && billingSettingsCompleted)
         || phoneNumberVerified"
       variant="success"
       @hidden="toastIsClosed"
       :visible="googleAccountConnectedOnce
         || merchantCenterAccountConnectedOnce
         || productFeedIsConfiguredOnce
         || freeListingIsActivatedOnce
-        || googleAdsAccountConnectedOnce
+        || (googleAdsAccountConnectedOnce && billingSettingsCompleted)
         || phoneNumberVerified"
```

Comment:
If you make the same schange two times, what do you think about having a computed?